tests/components/bmw_connected_drive/test_notify.py

```diff
@@ -0,0 +1,153 @@
+"""Test BMW numbers."""
+
+from unittest.mock import AsyncMock
+
+from bimmer_connected.models import MyBMWAPIError, MyBMWRemoteServiceError
+from bimmer_connected.tests.common import POI_DATA
+from bimmer_connected.vehicle.remote_services import RemoteServices
+import pytest
+import respx
+
+from homeassistant.core import HomeAssistant
+from homeassistant.exceptions import HomeAssistantError, ServiceValidationError
+
+from . import check_remote_service_call, setup_mocked_integration
+
+
+async def test_legacy_notify_service_simple(
+    hass: HomeAssistant,
+    bmw_fixture: respx.Router,
+) -> None:
+    """Test successful sending of POIs."""
+
+    # Setup component
+    assert await setup_mocked_integration(hass)
+
+    # Minimal required data
+    await hass.services.async_call(
+        "notify",
+        "bmw_connected_drive_ix_xdrive50",
+        {
+            "message": POI_DATA.get("name"),
+            "data": {
+                "latitude": POI_DATA.get("lat"),
+                "longitude": POI_DATA.get("lon"),
+            },
+        },
+        blocking=True,
+    )
+    check_remote_service_call(bmw_fixture, "send-to-car")
+
+    bmw_fixture.reset()
+
+    # Full data
+    await hass.services.async_call(
+        "notify",
+        "bmw_connected_drive_ix_xdrive50",
+        {
+            "message": POI_DATA.get("name"),
+            "data": {
+                "latitude": POI_DATA.get("lat"),
+                "longitude": POI_DATA.get("lon"),
+                "street": POI_DATA.get("street"),
+                "city": POI_DATA.get("city"),
+                "postal_code": POI_DATA.get("postal_code"),
+                "country": POI_DATA.get("country"),
+            },
+        },
+        blocking=True,
+    )
+    check_remote_service_call(bmw_fixture, "send-to-car")
+
+
+@pytest.mark.usefixtures("bmw_fixture")
+async def test_service_call_invalid_input(
+    hass: HomeAssistant,
+) -> None:
+    """Test invalid inputs."""
+
+    # Setup component
+    assert await setup_mocked_integration(hass)
+
+    with pytest.raises(ServiceValidationError):
```

Comment:
Once we put in the translation key we should test the error text to make sure the translation key was able to be resolved and formatted.